### PR TITLE
implement ConfirmationDelays() and KeyID() in ocr2vrf coordinator and…

### PR DESCRIPTION
**Pull request by jinhoonbang**
- implement ConfirmationDelays() and KeyID() in ocr2vrf coordinator 
- remove confirmationDelays from ocr2vrf job spec
- CLI script doesn't need KeyID for coordination setConfig anymore

### CODE REVIEW

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go [hunk 1]
```diff
@@ -588,6 +589,74 @@ func TestCoordinator_ReportIsOnchain(t *testing.T) {
 
 }
 
+func TestCoordinator_ConfirmationDelays(t *testing.T) {
+	t.Run("valid output", func(t *testing.T) {
+		expected := [8]uint32{1, 2, 3, 4, 5, 6, 7, 8}
+		ret := [8]*big.Int{}
+		for i, delay := range expected {
+			// assuming little endian
```
Comment (makramkd) [inline core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go:597]: Not sure what you mean by this comment - `big.Int` is big-endian (and so is the EVM) but I don't think it's relevant to this test right?

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go [hunk 2]
```diff
@@ -588,6 +589,74 @@ func TestCoordinator_ReportIsOnchain(t *testing.T) {
...
+		coordinatorContract.
+			On("SKeyID", mock.Anything).
+			Return(keyIDBytes, nil)
+		defer coordinatorContract.AssertExpectations(t)
+		c := &coordinator{
+			coordinatorContract: coordinatorContract,
+		}
+		keyID, err := c.KeyID(context.TODO())
+		assert.NoError(t, err)
+		assert.ElementsMatch(t, expected[:], keyID[:])
```
Comment (makramkd) [inline core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go:642]: I believe that this should be `assert.Equal` and not `assert.ElementsMatch` - the latter ignores the order of elements, whereas in this case we care about the order of the bytes in the byte array (different order implies a different key ID).

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go (hunk 1 above)
Comment (jinhoonbang) [reply core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go:597]: Ah this was a copy and paste error :p will remove.

File: core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go (hunk 2 above)
Comment (jinhoonbang) [reply core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator_test.go]: I see. Thanks for catching that. Updated

**Review by makramkd — APPROVED** [review]

**Pull request merged**